source/slang/slang-parser.cpp

```diff
@@ -7469,6 +7568,10 @@ namespace Slang
                 }
                 return newExpr;
             }
+            else if (identifierTokenContent == toSlice("let"))
```

Comment:
I don't think we need to make let an expr, since that could introduce ambiguities in places we don't want.

Reply:
done